test/ProjectTemplates/Microsoft.Extensions.AI.Templates.IntegrationTests/Snapshots/aichatweb.AzureOpenAI_Qdrant_Aspire.verified/aichatweb/aichatweb.Web/ResponseClientHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.Extensions.AI;
+using Aspire.OpenAI;
+using OpenAI;
+
+//namespace chat.aspire.Web;
+namespace Microsoft.Extensions.Hosting;
```

Comment:
Based on [other files](https://github.com/dotnet/extensions/blob/main/src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/ChatWithCustomData-CSharp.Web/Services/IngestedChunk.cs#L3).

```suggestion
namespace ChatWithCustomData_CSharp.Web;
```